src/cloud-api-adaptor/install/overlays/docker/kustomization.yaml

```diff
@@ -18,6 +18,12 @@ configMapGenerator:
     literals:
       - CLOUD_PROVIDER="docker"
       - CLOUD_CONFIG_VERIFY="false" # It's better set as true to enable could config verify in production env
+    #- DOCKER_HOST="unix:///var/run/docker.sock" # Uncomment and set if you want to use a specific docker host
+    #- DOCKER_API_VERSION="1.40" # Uncomment and set if you want to use a specific docker api version
```

Comment:
Nit: IIRC the minimum API version supported is 1.44? Is that always true, or could we write to an older one and 1.44 is the just the default? If 1.44 is always required, should we bump this version in the comment and add the restriction to the comment?

Reply:
Yeah this needs to be bumped